lips/lip-1.mdx

```diff
@@ -492,84 +451,234 @@ Represents a national ID.
 ```
 
 ### StatusEnum
-Valid values are:
+
+Valid values are the unicode strings:
 * `none` - No status is yet set from this actor.
 * `needs_kyc_data` - KYC data about the subaddresses is required by this actor.
-* `needs_recipient_signature` - Can only be associated with the sender actor.  Means that the sender still requires that the recipient VASP provide the signature so that the transaction can be put on-chain.
 * `ready_for_settlement` - Transaction is ready for settlement according to this actor (i.e. the required signatures/KYC data have been provided)
 * `abort` - Indicates the actor wishes to abort this payment, instead of settling it.
-* `pending_review` - Payment is pending review.
-* `soft_match` - Actor's KYC data resulted in a soft-match. The VASP associated with this actor will need to determine what next steps are required, and if it decides to send additional KYC information to clear the soft-match does so via the KYCObject field of additional_kyc_data. If not sent within SLA window, this transaction will be aborted.
+* `soft_match` - Actor's KYC data resulted in a soft-match.
+
+# PaymentObject Protocol Command Sequence
+
+**PaymentObject State.** The state of a `PaymentObject` is used to determine which VASP is expected to issue a command next, and what information is expected to be included in the command to progress the payment. The state is determined by the tuple of the status of the Sender and Receiver Actors and two flags `Sx` and `Rx`. The exact fields in the payment object for Sender and Receiver actor status are `sender.status.status` and `receiver.status.status`. The flag `Sx` and `Rx` are set when the Sender or the Receiver has included `additional_kyc_data` in the actor field (`Sender.kyc_data.additional_kyc_data` and `Receiver.kyc_data.additional_kyc_data` respectively).
+
+The states (Sender Status, Receiver Status, Sx Flag, Rx Flag) are:
```

Comment:
Should we replace Sx and Rx references with just Sender.kyc_data.additional_kyc_data and Receiver.kyc_data.additional_kyc_data?  It feels a bit confusing to me to call them flags when they really aren't and there is no such thing as Sx and Rx they are just concepts

Reply:
or `sender_set_additional_kyc` and `receiver_set_additional_kyc` in literal words